sources/packages/forms/src/form-definitions/institutionlocation.json

```diff
@@ -2372,6 +2308,79 @@
             "addons": [],
             "id": "efsn31k"
         },
+        {
+            "label": "clientType",
+            "labelWidth": "",
+            "labelMargin": "",
+            "customClass": "",
+            "modalEdit": false,
+            "defaultValue": null,
+            "persistent": "client-only",
+            "protected": false,
+            "dbIndex": false,
+            "encrypted": false,
+            "redrawOn": "data",
+            "customDefaultValue": "",
+            "calculateValue": "value = data.clientType;",
```

Comment:
Do we need to do this manual association here?

Reply:
you are right, we may not need it